prime-angular-frontend/src/app/shared/components/contact-information-form/contact-information-form.component.html

```diff
@@ -69,16 +72,16 @@
 
           </div>
 
-          <ng-container *ngIf="showSmsPhone">
+          <ng-container *ngIf="!forPhsa">
```

Comment:
If this is only used for PHSA you'll want to project the markup into the component and use a selector - `<ng-content select="..."></ng-content>`